splunk_add_on_ucc_framework/commands/rest_builder/validator_builder.py

```diff
@@ -68,10 +68,8 @@ def _get_class_name(self) -> str:
 
     def _get_arguments(self, config: Dict[str, Any]) -> Dict[str, Any]:
         ranges = config.get("range", [None, None])
-        return {
-            "max_val": ranges[1],
-            "min_val": ranges[0],
-        }
+        is_integer = config.get("isInteger", False)
+        return {"max_val": ranges[1], "min_val": ranges[0], "is_integer": is_integer}
```

Comment:
is it a backend validation or FE?

Reply:
backend validation

Reply:
For Python 3.7, it looks like this:
![image](https://github.com/user-attachments/assets/5b720a72-d2ce-4106-8548-be65472b3f1f)

code that is responsible for back-end validation: [Number().validate()](https://github.com/splunk/addonfactory-ucc-library/blob/main/splunktaucclib/rest_handler/endpoint/validator.py#L295-L303)

Reply:
@rohanm-crest if trailing zeros are not allowed on BE, should we restrict that from FE as well?

Reply:
Considering the below factors:
- these are language features that we are leveraging
- With FE + BE validations, the user won't be able to save these values. Just the error messages would be different but with the same meaning.

In this case, is it required to update the front-end to match the backend logic?